device.go

```diff
@@ -421,6 +444,91 @@ func getSCSIDevPath(scsiAddr string) (string, error) {
 	return filepath.Join(devPrefix, scsiDiskName), nil
 }
 
+// checkCCWBusFormat checks the format for the ccw bus. It needs to be in the form 0.<n>.<dddd>
+// n is the subchannel set ID - integer from 0 up to 3
+// dddd is the device id - integer in hex up to 0xffff
+// See https://www.ibm.com/support/knowledgecenter/en/linuxonibm/com.ibm.linux.z.ldva/ldva_r_XML_Address.html
+func checkCCWBusFormat(bus string) error {
+	busFormat := strings.Split(bus, ".")
+	if len(busFormat) != 3 {
+		return fmt.Errorf("Wrong bus format. It needs to be in the form 0.<n>.<dddd>, got %s", bus)
+	}
+
+	bus0, err := strconv.ParseInt(busFormat[0], 10, 32)
+	if err != nil {
+		return err
+	}
+	if bus0 != 0 {
+		return fmt.Errorf("Wrong bus format. First digit needs to be 0 instead is %d", bus0)
+	}
+
+	bus1, err := strconv.ParseInt(busFormat[1], 10, 32)
+	if err != nil {
+		return err
+	}
+	if bus1 > 3 {
```

Comment:
Could you define a variable to avoid hard-coding the number in the error message. Something like this (in-function `const`):

```go
const expectedMaxValue = 3
```